app/scripts/views/home/home-controller.js

```diff
@@ -28,6 +29,10 @@
             ctl.onSearchClicked = onSearchClicked;
             ctl.onTypeaheadSelected = onTypeaheadSelected;
             ctl.search = search;
+            ctl.map = null;
```

Comment:
No reason to put this on the scope, `map` should probably just be a private variable in the controller's closure.